src/adapter/src/coord/metrics.rs

```diff
@@ -13,8 +13,10 @@ use mz_ore::metric;
 use mz_ore::metrics::MetricsRegistry;
 use mz_sql::ast::{AstInfo, Statement, StatementKind};
 
+#[derive(Clone, Debug)]
 pub struct Metrics {
     pub query_total: IntCounterVec,
+    pub query_status: IntCounterVec,
```

Comment:
Does `query_total` do anything now? It seems like it's now a `sum` of `query_status`.

Reply:
the only difference is that `query_total` is incremented when the query is started and `query_status` is incremented when the query finishes - i wasn't sure if people cared about that behavior. if you don't think it makes a difference either way, i could collapse them into one.